### ISSUE TITLE
Forward options to the image node

### ISSUE THREAD

**Issue by azmeuk** [tushuhei/sphinxcontrib-screenshot#47]
The [image directive](https://docutils.sourceforge.io/docs/ref/rst/directives.html#image) has some interesting options that cannot be used by screenshots directive (alt text, alignment, etc.)

It would be nice if somehow sphinxcontrib-screenshot could use the image directive options.

There might be conflicting options like `:width:` or `:height:`. I am not sure what to do about these.

**Comment by azmeuk** [comment]
Maybe `:width:` and `:height:` could be renamed to `:viewport_width:` and `:viewport_height:` to match playwright naming and avoid conflict with `.. image::` options ?
